everything-true/everything-true.js

```diff
@@ -0,0 +1,11 @@
+function truthCheck(collection, pre) {
+  // Is everyone being true?
+  for (let obj of collection) {
+    if (!obj[pre]) {
+      return false;
```

Comment:
Have you read why this works https://guide.freecodecamp.org/javascript/falsy-values/